utils/__init__.py

```diff
@@ -0,0 +1 @@
+from .create_metrics import create_metrics
```

Comment:
nice - I am liking this style. I recently added it to cytomining/pycytominer#56 :tada: